perf_sim/main.cpp

```diff
@@ -1,8 +1,6 @@
 /**
- * main.cpp
- * Start point of the scalar MIPS CPU simulator.
- * MIPT-MIPS Assignment 4.
- * Ladin Oleg.
+ * main.cpp - entry point of the scalar MIPS CPU simulator
+ * Copyright 2017 MIPT-MIPS
```

Comment:
Cause I'm not sure I'm right with this, but
1. We have substituted the perf_sim implementation with that of Viktor
2. That particular file `main.cpp` used to have parameters parsing code, but now it contains nothing from the original one.